tests/Stripe/BaseStripeClientTest.php

```diff
@@ -202,4 +207,210 @@ public function testRequestWithOptsInParamsWarns()
         static::assertNotNull($charge);
         static::assertSame('acct_456', $this->optsReflector->getValue($charge)->headers['Stripe-Account']);
     }
+
+    public function testJsonRawRequestGetWithURLParams()
+    {
+        $curlClientStub = $this->getMockBuilder(\Stripe\HttpClient\CurlClient::class)
+            ->setMethods(['executeRequestWithRetries'])
+            ->getMock()
+        ;
+        $curlClientStub->method('executeRequestWithRetries')
+            ->willReturn(['{}', 200, []])
+        ;
```

Comment:
nit: interesting formatting.

Reply:
Oh weird, I guess it's a `PHP-CS-Fixer` thing. Same formatting in [StripeTelemetryTest](https://github.com/stripe/stripe-php/blob/6357a735/tests/Stripe/StripeTelemetryTest.php#L43)